pkg/applicationmanager/controllers/applicationrestore.go

```diff
@@ -1603,10 +1603,24 @@ func (a *ApplicationRestoreController) applyResources(
 	namespacedName := types.NamespacedName{}
 	namespacedName.Namespace = restore.Namespace
 	namespacedName.Name = restore.Name
-
-	pvNameMappings, err := a.getPVNameMappings(restore, objects)
-	if err != nil {
-		return err
+	// The applyResources is getting called in both the volume stage and resource stage.
+	// In the volume stage, it is getting called for applying the preRestore object.
+	// During the volume stage, we will not have restoreVolume updated in the volumeInfo structure.
+	// In between two driver's PVC restore processing, there is a chance that applicationrestore CR will status.VolumeInfo
+	// updated with the basic information of the volume, with out restoreVolume name.
+	// List of prerestore resource for each driver is as follow:
+	// aws, azure, gke driver does have any preRestore object.
```

Comment:
type: should it be does not have any preRestore Object?

Reply:
Fixed